src/dex/verified/constants.ts

```diff
@@ -0,0 +1,10 @@
+export const MIN_USD_LIQUIDITY_TO_FETCH = 100n;
```

Comment:
@Mohzcrea8me are we using these constants ?

Reply:
No for now, but I think to fetchAllPools we need liquidity requirement like balancer did I noticed most of the pools does not have liquidity so I removed the minimum liquidity.